meos/include/meos_internal.h

```diff
@@ -287,6 +287,49 @@ extern void stbox_set_box3d(const STBox *box, BOX3D *box3d);
  * Functions for temporal types
  *****************************************************************************/
 
+/* Macros for speeding up access to components of temporal sequences (sets)*/
+
+#ifdef DEBUG_BUILD
+extern const TInstant *tsequence_inst_n(const TSequence *seq, int index);
+extern const TSequence *tsequenceset_seq_n(const TSequenceSet *ss, int index);
```

Comment:
It seems we do not define _tsequence_offset_ptr_ as extern in the debug build.
Is this on purpose?
If so, maybe set the function to static when defining it.